src/pages/Positions.tsx

```
@@ -1,26 +1,716 @@
+
+      switch (sortField) {
+        case 'borrowedAmount':
+          aValue = parseFloat(a.initialBorrowQuote);
+          bValue = parseFloat(b.initialBorrowQuote);
+          break;
+        case 'interest':
+          aValue = a.interestAccrued;
+          bValue = b.interestAccrued;
+          break;
+        case 'riskLevel':
+          const riskOrder = { low: 1, medium: 2, high: 3 };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)